### DISCUSSION TITLE
Step by step instructions to install MMEX on Synology NAS

### DISCUSSION THREAD

**Discussion by tycariadus** [moneymanagerex/moneymanagerex#6382] [Q&A]
I have a NAS drive which is fairly new to me, and want to look at how to install MMEX so that we are in control of the data location, and to be able to access from say a phone.  I see that there are various bits of advice but not step by step instructions.  Has anyone written any?  Thanks!

**Comment by whalley** [comment]
Have you tried this? https://www.synoforum.com/threads/moneymanager-ex-running-a-linux-image-on-synology-via-docker.12076/post-60389

**Comment by PMaff** [comment]
May I ask, why you want to run it on the NAS and not on the connected desktop machine (Linux, Windows, MAC)?

**Comment by tycariadus** [comment]
Hi Peter:

 

So we can access the app on say a mobile phone.

**Comment by DevAnyKR** [comment]
I use OneDrive.
Synology Drive syncs in the same way; it's ideal for DB file sharing.
You must use the Drive utility, so that you can mount it to a folder.
